src/main/java/view/SearchPanel.java

```diff
@@ -0,0 +1,134 @@
+package view;
+
+import use_case.note.search.SearchInteractor;
+import interface_adapter.PlayGameController;
+import use_case.note.FetchPlayerStatisticsInputBoundary; // Ensure this import is correct
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SearchPanel {
+    private final SearchInteractor interactor;
+    private final JFrame frame;  // Make the frame a class variable for reuse
+
+    public SearchPanel(SearchInteractor interactor) {
+        this.interactor = interactor;
+        this.frame = new JFrame("User Search Application");
+        createAndShowGUI();
+    }
+
+    private void createAndShowGUI() {
+        // Configure the main frame
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        frame.setSize(500, 350);
+        frame.setLayout(new BorderLayout());
+        frame.setLocationRelativeTo(null); // Center the window on the screen
+
+        // Create the input panel
```

Comment:
The input panel is clear, but consider extracting its creation into a helper method for clarity.